src/plugins/intel_npu/src/backend/src/zero_device.cpp

```diff
@@ -23,6 +23,17 @@ ZeroDevice::ZeroDevice(const std::shared_ptr<ZeroInitStructsHolder>& initStructs
     zeroUtils::throwOnFail("zeDeviceGetProperties",
                            zeDeviceGetProperties(_initStructs->getDevice(), &device_properties));
 
+    // Query activity for calculating device utilization
+    uint32_t engine_count = 0;
+    zeroUtils::throwOnFail("zesDeviceEnumEngineGroups",
+                           zesDeviceEnumEngineGroups(_initStructs->getDevice(), &engine_count, nullptr));
```

Comment:
You need to add this zesDeviceEnumEngineGroups to https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_npu/src/utils/include/intel_npu/utils/zero/zero_api.hpp#L20 as well.

Reply:
Also, is this supported by NPU drivers?

Reply:
It looks like linux drivers return some errors. Maybe because it is not implemented there?

Reply:
Also, we need to check if it was supported with older drivers.

Reply:
Is there a way to not return a throw if it is not implemented but just a default value?

Reply:
enabled try-catch code here to return a default value if this property is not implemented or other issues from the driver.

Reply:
But the problem is actually here. zesDeviceEnumEngineGroups is not supported by the driver and it will fail. Please check the return code properly and set a default value if returns that it is not supported.